src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -304,6 +314,12 @@ export default Vue.extend({
             this.powerSaveBlocker = null
           }
         })
+
+        const _this = this
+        this.player.textTrackSettings.on('modalclose', function () {
+          const settings = this.getValues()
+          _this.updateDefaultCaptionSettings(JSON.stringify(settings))
+        })
```

Comment:
I'd prefer it if you did this instead:
```suggestion
        this.player.textTrackSettings.on('modalclose', (_) => {
          const settings = this.player.textTrackSettings.getValues()
          this.updateDefaultCaptionSettings(JSON.stringify(settings))
        })
        // OR
        this.player.textTrackSettings.on('modalclose', (event) => {
          const settings = event.target.getValues()
          this.updateDefaultCaptionSettings(JSON.stringify(settings))
        })
```

Reply:
Thank you for the suggestion! It has been fixed in the most recent commit.